### PR TITLE
feat: added alert components

### PR DESCRIPTION
**Pull request by anamkaa**
Screenshot (219)

Screenshot (220)

### CODE REVIEW

File: stylesheets/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@import url("/stylesheets/main.css");
+
+body{
+    background-color: #f8f8ff;
```
Comment (tanishq20) [inline stylesheets/components/alert.css]: You can use var() to give background color.

**Review by tanishq20 — COMMENTED** [review]
Just one small change.

File: stylesheets/components/alert.css (hunk 1 above)
Comment (anamkaa) [reply stylesheets/components/alert.css]: I've done as suggested , thankyou.

**Comment by aishwarya-bm** [comment]
So many variations, looks good to merge!

**Pull request merged**